docs/_docs/upgrading.md

```diff
@@ -8,3 +8,6 @@ Upgrading from an older version of Jekyll? Upgrading to a new major version of J
 
 - [From 0.x to 1.x and 2.x](/docs/upgrading/0-to-2/)
 - [From 2.x to 3.x](/docs/upgrading/2-to-3/)
+
+If you are making a minor update (for example from 3.3.1 to the latest version at the time 3.3.2) run 'bundle update jekyll' when in your site directory. 
```

Comment:
Do you have suggestted text, that would not suggest for anyone to try it for something like 3.9.7 to 4.0.0? I wanted the impression a new person should check in those cases without the need for them to understand semantic versioning already ? I don't want to relay on all breaking changes occuring with documentation. As most documentation tends to trail behind.